tests/by-util/test_numfmt.rs

```diff
@@ -2,6 +2,14 @@
 
 use crate::common::util::*;
 
+#[test]
+fn test_should_not_round_floats() {
+    new_ucmd!()
+        .args(&["1.1", "1.22", ".1", "-0.1"])
```

Comment:
can you add some examples like 0.99 and 1.01 to check these behavior too?

Reply:
Ok, added.